tests/QueueTest.php

```diff
@@ -149,7 +149,7 @@ public function testQueueRelease(): void
     {
         Carbon::setTestNow();
         $queue = 'test';
-        $delay = 123;
+        $delay = now()->addSeconds(123);
```

Comment:
This caused the tests to fail so sneaking it in with these other changes
https://github.com/stubbo/laravel-mongodb/actions/runs/7492205173/job/20395216401

Reply:
The issue may be that `Carbon::setTestNow()` must receive a date as argument. As it is line 151, the clock is reset to real time.
https://github.com/briannesbitt/Carbon/blob/add90c4558af9bda13fcdaec10ce137c3a3e300b/readme.md?plain=1#L30C1-L39

Reply:
Thanks for mentioning that, I've not done much with tests so nice to know that exists :)